### PR TITLE
Dedup cities in parsed data

### PR DESCRIPTION
**Pull request by drwestco**
Raw data doesn't have consistent case in the Cities field, so we can wind up with duplicate entries in the rendered list.

### CODE REVIEW

File: public/toDataByLocation.js [hunk 1]
```diff
@@ -3,14 +3,13 @@ export default function toDataByLocation(data) {
   const approvedIndex = headers.findIndex( e => e === 'approved' );
   const stateIndex = headers.findIndex( e => e === 'state' );
   const cityIndex = headers.findIndex( e => e === 'city' );
-  const instructionsIndex = headers.findIndex( e => e === 'instructions' );
```
Comment (graemer) [inline public/toDataByLocation.js]: Why is this removed?

Comment (drwestco) [reply public/toDataByLocation.js]: Cleanup from previous checkin. Linkify code used to be done here; now this value is unused.
